pom.xml

```diff
@@ -20,19 +20,32 @@
     <modelVersion>4.0.0</modelVersion>
     <parent>
         <groupId>nl.knaw.dans.shared</groupId>
-        <artifactId>dans-prototype</artifactId>
+        <artifactId>dans-scala-prototype</artifactId>
         <version>1.32</version>
     </parent>
     <groupId>nl.knaw.dans.easy</groupId>
     <artifactId>easy-schema</artifactId>
     <name>EASY Schema Definitions</name>
     <description>Collection of metadata schema's used by EASY</description>
     <version>1.x-SNAPSHOT</version>
-    <inceptionYear>2015-2016</inceptionYear>
+    <inceptionYear>2015-2017</inceptionYear>
```

Comment:
I personally doubt we should make changes like this. IMO we should stick to a single inception year and keep it at that. So, `inceptionYear` should be `2015` in this case. Of course I'm not a lawyer, and maybe we should discuss this sort of yearly change with Heiko, but it seems a bit silly to me!